browsers/webview_ios.json

```
@@ -0,0 +1,302 @@
+          "release_date": "2023-01-23",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-16_3-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "614.4.6"
+        },
+        "16.4": {
+          "release_date": "2023-03-27",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-16_4-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "615.1.26"
+        },
+        "16.5": {
+          "release_date": "2023-05-18",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-16_5-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "615.2.9"
+        },
+        "16.6": {
+          "release_date": "2023-07-24",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-16_6-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "615.3.12"
+        },
+        "17": {
+          "release_date": "2023-09-18",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-17-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "616.1.27"
+        },
+        "17.1": {
+          "release_date": "2023-10-25",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-17_1-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "616.2.9"
+        },
+        "17.2": {
+          "release_date": "2023-12-11",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-17_2-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "617.1.17"
+        },
+        "17.3": {
+          "release_date": "2024-01-22",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-17_3-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "617.2.4"
+        },
+        "17.4": {
+          "release_date": "2024-03-05",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-17_4-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "618.1.15"
+        },
+        "17.5": {
+          "release_date": "2024-05-13",
+          "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-17_5-release-notes",
+          "status": "retired",
+          "engine": "WebKit",
+          "engine_version": "618.2.12"
+        },
+        "17.6": {
+          "release_date": "2024-07-22",
```

We just corrected it this morning in the other file :)

```suggestion
          "release_date": "2024-07-29",
```